reactor-core/src/main/java/reactor/core/scheduler/BoundedElasticPerThreadScheduler.java

```diff
@@ -22,10 +22,10 @@
 import reactor.core.Disposable;
 import reactor.core.Scannable;
 
-final class ThreadPerTaskBoundedElasticScheduler
-		implements Scheduler, SchedulerState.DisposeAwaiter<ThreadPerTaskBoundedElasticScheduler.BoundedServices>, Scannable {
+final class BoundedElasticPerThreadScheduler
+		implements Scheduler, SchedulerState.DisposeAwaiter<BoundedElasticPerThreadScheduler.BoundedServices>, Scannable {
 
-	ThreadPerTaskBoundedElasticScheduler(int maxThreads, int maxTaskQueuedPerThread, ThreadFactory factory) {
+	BoundedElasticPerThreadScheduler(int maxThreads, int maxTaskQueuedPerThread, ThreadFactory factory) {
```

Comment:
Similarly here, something like:

"This `BoundedElasticPerThreadScheduler` variant is included when Reactor is used with JDK versions lower than 21, and all methods raise an UnsupportedOperationException. An alternative variant is available for use on JDK 21+ where virtual threads are supported".